kubernetes-client/src/main/java/io/fabric8/kubernetes/client/utils/ExponentialBackoffIntervalCalculator.java

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (C) 2015 Red Hat, Inc.
```

Comment:
Should I update the year? 2015 -> 2021?

Reply:
The license header check I believe expects 2015.